tapdb/sqlc/migrations/000007_universe.up.sql

```diff
@@ -85,3 +85,52 @@ CREATE VIEW universe_stats AS
     FROM universe_events u
     JOIN universe_roots roots ON u.universe_root_id = roots.id
     GROUP BY roots.asset_id, roots.group_key, roots.namespace_root;
+
+-- This table contains global configuration for universe federation syncing.
+CREATE TABLE IF NOT EXISTS federation_global_sync_config (
+    -- This field is an enum representing the proof type stored in the given
+    -- universe.
+    proof_type TEXT NOT NULL PRIMARY KEY CHECK(proof_type IN ('issuance', 'transfer')),
+
+    -- This field is a boolean that indicates whether or not a universe of the
+    -- given proof type should accept remote proof insertion via federation
+    -- sync.
+    allow_sync_insert BOOLEAN NOT NULL,
+
+    -- This field is a boolean that indicates whether or not a universe of the
+    -- given proof type should accept remote proof export via federation sync.
+    allow_sync_export BOOLEAN NOT NULL
+);
+
+-- This table contains universe (asset/asset group) specific federation sync
+-- configuration.
+CREATE TABLE IF NOT EXISTS federation_uni_sync_config (
```

Comment:
IIUC, in order for something to later on have a foreign key that refs this table, it _does_ in fact still need a primary key. 

I think we just want a composite primary key here. Though I saw you had to re-work things slightly tho, non-blocking comment.